bdb/bdb_api.h

```diff
@@ -135,29 +135,39 @@ struct bdb_queue_stats {
 };
 
 /* This is identical to bb_berkdb_thread_stats in db.h */
+
+/* Multiplication usually takes fewer CPU cycles than division. Therefore
+   when comparing a usec and a msec, it is preferable to use:
+   usec <comparison operator> USEC(msec)  */
+#ifndef MSEC
+#define MSEC(usec) ((usec) / 1000)
```

Comment:
Suggestion: May it help if we rename this USEC_to_MSEC so its immediately clear what it does? (and USEC_TO_MSEC() below)